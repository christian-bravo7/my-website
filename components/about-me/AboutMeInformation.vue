<template>
  <section class="section home__content--secondary min-screen who-iam-section">
    <div class="who-iam-section__container container">
      <div class="who-iam-section__content is-medium">
        <h1 class="who-iam-section__title">
          {{ $t('me.i-label') }}
        </h1>
        <p class="who-iam-section__description">
          {{ $t('me.about-me-description') }}
        </p>
      </div>
      <div class="who-iam-section__asset">
        <RemolinoLogo />
      </div>
    </div>
  </section>
</template>

<script lang="ts">
import { Vue, Component } from 'nuxt-property-decorator';

import RemolinoLogo from '~/assets/img/remolino.svg?inline';

@Component({
  components: {
    RemolinoLogo,
  },
})
export default class AboutMeInformation extends Vue {}

</script>

<style lang="scss" scoped>
.who-iam-section {
  display: flex;
  align-items: center;

  &__container {
    display: grid;
    grid-template-columns: 1fr 1fr;
    color: var(--text);
  }

  &__content {
    position: relative;
  }

  &__description {
    position: relative;
    z-index: 2;
    margin-top: rem(120);
    font-size: rem(18);
    line-height: 2;
  }

  &__title {
    position: absolute;
    top: 0;
    width: 100%;
    color: var(--asset-highlight);
    font-weight: bold;
    font-size: rem(182);
    line-height: 1;
    opacity: 0.5;
  }

  &__asset {
    display: flex;
    align-items: center;
    justify-content: center;

    & > svg {
      width: 90%;
      max-height: rem(300);
      fill: var(--asset-highlight);
    }
  }
}

.home {
  display: grid;
  grid-template-rows: 1fr;
  grid-template-columns: 1fr;
  align-items: center;
  justify-content: center;
  background-color: var(--background);

  &__content {
    padding: rem(92) rem(24);
    background-color: var(--background);

    &--secondary {
      background-color: var(--text-light);
    }

    &--tertiary {
      background-color: var(--background-footer);
    }
  }
}

@include mobile {
  .who-iam-section {
    &__container {
      grid-template-columns: 1fr;
    }

    &__asset {
      display: none;
    }
  }
}

</style>
