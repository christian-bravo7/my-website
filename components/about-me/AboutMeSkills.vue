<template>
  <section class="section min-screen skills-section">
    <div class="skills-section__container container">
      <h2 class="has-text-centered skills-section__title">
        {{ $t('me.skills-label') }}
      </h2>
      <div class="skills-section__collection">
        <SkillIcon name="HTML5">
          <HtmlIcon />
        </SkillIcon>
        <SkillIcon name="CSS3">
          <CssIcon />
        </SkillIcon>
        <SkillIcon name="SASS">
          <SassIcon />
        </SkillIcon>
        <SkillIcon name="Nuxt.js">
          <NuxtIcon />
        </SkillIcon>
        <SkillIcon name="React.js">
          <ReactIcon />
        </SkillIcon>
        <SkillIcon name="Vue.js">
          <VueIcon />
        </SkillIcon>
        <SkillIcon name="Typescript">
          <TypescriptIcon />
        </SkillIcon>
        <SkillIcon name="Javascript">
          <JavascriptIcon />
        </SkillIcon>
        <SkillIcon name="Node.js">
          <NodeIcon />
        </SkillIcon>
        <SkillIcon name="GraphQL">
          <GraphqlIcon />
        </SkillIcon>
        <SkillIcon name="Git">
          <GitIcon />
        </SkillIcon>
        <SkillIcon name="Jest">
          <JestIcon />
        </SkillIcon>
      </div>
    </div>
  </section>
</template>

<script lang="ts">
import { Vue, Component } from 'nuxt-property-decorator';

import CssIcon from '~/assets/img/css3.svg?inline';
import JavascriptIcon from '~/assets/img/javascript.svg?inline';
import NuxtIcon from '~/assets/img/nuxt-dot-js.svg?inline';
import ReactIcon from '~/assets/img/react.svg?inline';
import VueIcon from '~/assets/img/vue-dot-js.svg?inline';
import TypescriptIcon from '~/assets/img/typescript.svg?inline';
import NodeIcon from '~/assets/img/node-dot-js.svg?inline';
import GraphqlIcon from '~/assets/img/graphql.svg?inline';
import GitIcon from '~/assets/img/git.svg?inline';
import SassIcon from '~/assets/img/sass.svg?inline';
import JestIcon from '~/assets/img/jest.svg?inline';
import HtmlIcon from '~/assets/img/html5.svg?inline';

@Component({
  components: {
    CssIcon,
    JavascriptIcon,
    NuxtIcon,
    ReactIcon,
    VueIcon,
    TypescriptIcon,
    GraphqlIcon,
    GitIcon,
    SassIcon,
    JestIcon,
    HtmlIcon,
    NodeIcon,
  },
})
export default class AboutMeSkills extends Vue {}

</script>

<style lang="scss" scoped>
.skills-section {
  display: flex;
  align-items: center;

  &__container {
    display: grid;
    grid-template-columns: 1fr;
  }

  &__title {
    margin-bottom: rem(24);
    color: var(--text-light);
    font-weight: bold;
    font-size: rem(182);
    word-break: break-word;
  }

  &__collection {
    display: grid;
    grid-gap: rem(25);
    grid-template-columns: repeat(auto-fill, minmax(150px, 1fr));
    justify-items: center;
    padding: rem(24) 0;
  }
}

.home {
  &__content {
    padding: rem(92) rem(24);
    background-color: var(--background);

    &--secondary {
      background-color: var(--text-light);
    }

    &--tertiary {
      background-color: var(--background-footer);
    }
  }
}

@include mobile {
  .skills-section {
    &__title {
      font-size: rem(70);
    }

    &__collection {
      grid-gap: rem(10) rem(5);
    }
  }
}
</style>
