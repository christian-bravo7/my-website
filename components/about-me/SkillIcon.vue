<template>
  <div class="flex flex-col group items-center justify-center w-full p-2 skill-icon hover:shadow-lg transition-shadow duration-150">
    <span class="w-16 text-pink-800 dark:text-blue-500">
      <slot name="default" />
    </span>
    <span class="mt-2 text-gray-300 dark:text-blue-700 group-hover:text-pink-800 dark-group-hover:text-blue-500 text-base md:text-lg font-bold transition-colors duration-150">
      {{ name }}
    </span>
  </div>
</template>

<script lang="ts">
import { Vue, Component, Prop } from 'nuxt-property-decorator';

@Component
export default class SkillIcon extends Vue {
  @Prop({ type: String, required: true })
  readonly name!: string;
}
</script>

<style lang="scss" scoped>
.skill-icon {
  svg {
    @apply fill-current;
  }
}
</style>
