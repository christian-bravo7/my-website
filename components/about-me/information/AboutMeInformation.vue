<template>
  <section class="section-container">
    <div class="container | flex | my-0 mx-auto">
      <div class="text-black dark:text-white | about-me-information__content">
        <div class="relative | mb-4 md:mb-0">
          <h2 class="mb-4 | section-title about-me-information__title">
            {{ $t('me.hi-label') }}
          </h2>
          <p
            class="relative z-10 | font-base | about-me-information__description"
            v-html="$md($t('me.about-me-description', { age: currentAge }))"
          />
        </div>
        <div class="flex items-center justify-center">
          <img
            src="~assets/img/photo-2.png"
            alt="christian"
          >
        </div>
      </div>
    </div>
  </section>
</template>

<script lang="ts">
import { Vue, Component } from 'nuxt-property-decorator';

@Component
export default class AboutMeInformation extends Vue {
  readonly birthday: string = '2000-01-15';

  get currentAge (): number {
    const currentDate = new Date();
    const birth = new Date(Date.parse(this.birthday));

    const currentMonth = currentDate.getMonth();
    const currentYear = currentDate.getFullYear();
    const currentDay = currentDate.getDate();
    const birthMonth = birth.getMonth();
    const birthYear = birth.getFullYear();
    const birthDay = birth.getDate();

    const age = currentYear - birthYear;
    const ageNotCompleted = age - 1;

    if (birthMonth > currentMonth) {
      return ageNotCompleted;
    }

    if (birthMonth < currentMonth) {
      return age;
    }

    if (birthDay > currentDay) {
      return ageNotCompleted;
    }

    return age;
  }
}
</script>

<style lang="scss" scoped>
.about-me-information {
  &__content {
    @apply grid grid-cols-1 items-center;
  }

  &__title {
    @include theme-gradients(four, five);
  }

  &__description {
    /deep/ p {
      @apply mb-4;
    }
  }
}

@screen lg {
  .about-me-information {
    &__content {
      @apply gap-40;

      grid-template-columns: 1fr auto;
    }
  }
}
</style>
