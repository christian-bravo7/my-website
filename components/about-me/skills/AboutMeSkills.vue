<template>
  <section class="section-container">
    <div class="container | flex | my-0 mx-auto">
      <div class="flex flex-col | w-full">
        <h2 class="mb-8 | text-center break-all | section-title about-me-skills__title">
          {{ $t('me.skills-label') }}
        </h2>
        <div class="flex justify-center items-center | flex-1">
          <div class="flex flex-wrap justify-center | max-w-screen-lg">
            <SkillIcon
              v-for="({ name, component }, index) in skills"
              :key="index"
              :name="name"
              class="m-2"
            >
              <Component :is="component" />
            </SkillIcon>
          </div>
        </div>
      </div>
    </div>
  </section>
</template>

<script lang="ts">
import { Vue, Component } from 'nuxt-property-decorator';

import CssIcon from '~/assets/img/css3.svg?inline';
import JavascriptIcon from '~/assets/img/javascript.svg?inline';
import NuxtIcon from '~/assets/img/nuxt-dot-js.svg?inline';
import ReactIcon from '~/assets/img/react.svg?inline';
import VueIcon from '~/assets/img/vue-dot-js.svg?inline';
import TypescriptIcon from '~/assets/img/typescript.svg?inline';
import NodeIcon from '~/assets/img/node-dot-js.svg?inline';
import GitIcon from '~/assets/img/git.svg?inline';
import SassIcon from '~/assets/img/sass.svg?inline';
import JestIcon from '~/assets/img/jest.svg?inline';
import HtmlIcon from '~/assets/img/html5.svg?inline';

@Component({
  components: {
    CssIcon,
    JavascriptIcon,
    NuxtIcon,
    ReactIcon,
    VueIcon,
    TypescriptIcon,
    GitIcon,
    SassIcon,
    JestIcon,
    HtmlIcon,
    NodeIcon,
  },
})
export default class AboutMeSkills extends Vue {
  skills: Array<{ name: string, component: any }> = [
    { name: 'Javascript', component: JavascriptIcon },
    { name: 'Typescript', component: TypescriptIcon },
    { name: 'NuxtJS', component: NuxtIcon },
    { name: 'VueJS', component: VueIcon },
    { name: 'ReactJS', component: ReactIcon },
    { name: 'NodeJS', component: NodeIcon },
    { name: 'Git', component: GitIcon },
    { name: 'Jest', component: JestIcon },
    { name: 'SASS', component: SassIcon },
    { name: 'CSS3', component: CssIcon },
    { name: 'HTML5', component: HtmlIcon },
  ]
}
</script>

<style lang="scss" scoped>
.about-me-skills {
  &__title {
    @extend .gradient-one;
  }
}

html.dark-mode {
  .about-me-skills {
    &__title {
      @extend .gradient-two;
    }
  }
}
</style>
