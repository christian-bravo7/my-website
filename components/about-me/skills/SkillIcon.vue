<template>
  <div class="flex flex-col items-center justify-between | p-2 | bg-gray-100 dark:bg-blue-700 | rounded-xl shadow-md hover:shadow-xl | select-none | group transition-shadow duration-300 | skill-icon">
    <span class="flex justify-center items-center flex-1 | w-16">
      <slot name="default" />
    </span>
    <span class="mt-2 | text-black dark:text-white group-hover:text-pink-500 dark-group-hover:text-blue-500 | text-base md:text-lg | font-bold | transition-colors duration-300">
      {{ name }}
    </span>
  </div>
</template>

<script lang="ts">
import { Vue, Component, Prop } from 'nuxt-property-decorator';

@Component
export default class SkillIcon extends Vue {
  @Prop({ type: String, required: true })
  readonly name!: string;
}
</script>

<style lang="scss" scoped>
.skill-icon {
  min-width: 140px;

  svg {
    @apply fill-current;
  }
}
</style>
