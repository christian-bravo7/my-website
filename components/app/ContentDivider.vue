<template>
  <section
    class="content-divider"
  >
    <div
      class="content-divider__section"
      :class="{ 'content-divider__section--hidden': second }"
    />
    <div
      class="content-divider__section"
      :class="{ 'content-divider__section--hidden': first }"
    />
  </section>
</template>

<script lang="ts">
import { Vue, Component, Prop } from 'nuxt-property-decorator';

@Component
export default class ContentDivider extends Vue {
  @Prop({ type: Boolean, default: false })
  readonly first!: boolean;

  @Prop({ type: Boolean, default: false })
  readonly second!: boolean;
}
</script>

<style lang="scss" scoped>
.content-divider {
  display: grid;
  grid-template-columns: 1fr 1fr;
  height: rem(80);

  &__section {
    height: 100%;
    background-color: var(--text-light);

    &--hidden {
      opacity: 0;
    }
  }
}

@include mobile {
  .content-divider {
    height: rem(40);
  }
}
</style>
