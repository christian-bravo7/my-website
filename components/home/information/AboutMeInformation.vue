<template>
  <section class="section-container">
    <div class="container | flex justify-center | my-0 mx-auto">
      <div class="max-w-screen-lg | about-me-information__content">
        <div class="mb-4 md:mb-0">
          <h2 class="mb-8 | section-title about-me-information__title">
            {{ $t('me.about-me-label') }}
          </h2>
          <p
            class="font-base | about-me-information__description"
            v-html="$md($t('me.about-me-description'))"
          />
        </div>
        <div class="flex items-center justify-center">
          <img
            src="~assets/img/photo-2.png"
            alt="christian picture"
          >
        </div>
      </div>
    </div>
  </section>
</template>

<script lang="ts">
import { Vue, Component } from 'nuxt-property-decorator';

@Component
export default class AboutMeInformation extends Vue {}
</script>

<style lang="scss" scoped>
.about-me-information {
  &__content {
    @apply grid grid-cols-1 items-center;
  }

  &__title {
    @include theme-gradients(four, five);
  }

  &__description {
    /deep/ p {
      @apply mb-4;
    }
  }
}

@screen lg {
  .about-me-information {
    &__content {
      @apply gap-40;

      grid-template-columns: 2fr 1fr;
    }
  }
}
</style>
