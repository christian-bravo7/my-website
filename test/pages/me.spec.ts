import { createLocalVue, mount, shallowMount } from '@vue/test-utils';
import Me from '@/pages/me.vue';
import PortalVue from 'portal-vue';
import VueMeta from 'vue-meta';
import ContentDivider from '@/components/app/ContentDivider.vue';
import AboutMeInformation from '@/components/about-me/AboutMeInformation.vue';
import AboutMeSkills from '@/components/about-me/AboutMeSkills.vue';
import SkillIcon from '@/components/about-me/SkillIcon.vue';

const localVue = createLocalVue();
localVue.use(PortalVue);
localVue.use(VueMeta, { keyName: 'head' });

describe('me page', () => {
  it('should render correctly', () => {
    const aboutMeSkills = AboutMeSkills;

    // @ts-ignore
    aboutMeSkills.options.components.SkillIcon = SkillIcon;

    const wrapper = mount(Me, {
      components: {
        ContentDivider,
        AboutMeInformation,
        AboutMeSkills,
      },
      localVue,
      mocks: {
        $t: jest.fn(msg => msg),
        $scrollreveal: {
          reveal: jest.fn(() => {}),
          sync: jest.fn(() => {}),
        },
      },
    });

    expect(wrapper.html()).toMatchSnapshot();
  });

  it('should display meta-tag title', () => {
    const wrapper = mount(Me, {
      components: {
        ContentDivider,
        AboutMeInformation,
        AboutMeSkills,
      },
      localVue,
      mocks: {
        $t: jest.fn(msg => msg),
        $scrollreveal: {
          reveal: jest.fn(() => {}),
          sync: jest.fn(() => {}),
        },
      },
    });

    const title = wrapper.vm.$meta().refresh().metaInfo.title;

    expect(title).toBe('navigation.about-me-label');
  });
});
