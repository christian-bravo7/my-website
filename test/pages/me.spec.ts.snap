// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`me page should render correctly 1`] = `
<section>
  <div class="v-portal" style="display: none;"></div>
  <section class="section-container mb-24 md:mb-0">
    <div class="max-w-screen-lg | flex justify-center | my-0 mx-auto">
      <div class="about-me-information__content">
        <div class="mb-4 md:mb-0">
          <h2 class="mb-8 | section-title about-me-information__title">
            About me
          </h2>
          <p class="font-base | about-me-information__description">
          <p>Hello, my name is Christian Bravo, a Frontend Developer from Guatemala with a great passion for web development using modern technologies.</p>
          <p>I enjoy solving problems, creating dynamic and intuitive user interfaces, developing rich web applications and experiences.</p>
          <p>Outside of the office you will find me playing soccer, collecting cactus and spending time with my pets and family.</p>
          </p>
        </div>
        <div class="flex items-center justify-center"><img src="~assets/img/photo-2.png" alt="christian picture" class="w-5/6 md:w-auto"></div>
      </div>
    </div>
  </section>
  <section class="section-container mb-24 md:mb-0">
    <div class="max-w-screen-lg | flex justify-center items-center | w-full | my-0 mx-auto | overflow-x-hidden">
      <div class="flex flex-col | w-full">
        <div class="flex justify-center | mb-16">
          <h2 class="section-title my-work-experience__title">
            Where I've worked
          </h2>
        </div>
        <div class="flex flex-col md:flex-row">
          <div class="relative | my-work-experience__button-wrapper">
            <div class="flex md:flex-col | mb-8 md:mb-0 pb-4 md:pb-0 | overflow-x-scroll md:overflow-x-auto | my-work-experience__button-list"><button class="relative | px-4 py-4 | text-left | rounded-lg | text-black dark:text-white | whitespace-nowrap | transition-all duration-300 | my-work-experience-button my-work-experience-button--is-active">
                Telus International
              </button> <button class="relative | px-4 py-4 | text-left | rounded-lg | text-black dark:text-white | whitespace-nowrap | transition-all duration-300 | my-work-experience-button">
                Master Development
              </button> <button class="relative | px-4 py-4 | text-left | rounded-lg | text-black dark:text-white | whitespace-nowrap | transition-all duration-300 | my-work-experience-button">
                Viaro Networks
              </button></div>
          </div>
          <div class="flex-1 | md:ml-6 | my-work-experience__card-wrapper">
            <div>
              <h6 class="text-xl | mb-2"><span class="font-bold">
    Software Developer
  </span> <span>at</span> <a href="https://www.telusinternational.com" target="_blank" rel="noopener noreferrer" class="text-pink-500 dark:text-blue-500 | font-bold">
                  Telus International
                </a></h6>
              <div class="flex text-sm mb-6"><span><span class="capitalize">
      Nov 2020
    </span> <span>-</span> <span class="capitalize">
      present
    </span></span> <span class="flex items-center | mx-2"><span class="w-1 h-1 | rounded-full | bg-black dark:bg-white"></span></span> <span class="text-gray-700 dark:text-gray-300">
    6 months
  </span></div>
              <div class="mb-6">
                <div class="flex mb-2"><span class="material-icons">
      arrow_right
    </span> <span class="leading-loose | font-base">
      Development of new features, bug fixing and support for web applications.
    </span></div>
              </div>
              <div><span class="block | font-bold | text-xs | mb-6">
    Technologies:
  </span>
                <div class="flex flex-wrap">
                  <div class="mr-4 mb-4 py-2 px-4 | text-sm | font-bold | whitespace-nowrap | rounded-full | border border-pink-500 dark:border-blue-300 | bg-pink-50 bg-opacity-30 dark:bg-blue-700 dark:bg-opacity-25">
                    Angular
                  </div>
                  <div class="mr-4 mb-4 py-2 px-4 | text-sm | font-bold | whitespace-nowrap | rounded-full | border border-pink-500 dark:border-blue-300 | bg-pink-50 bg-opacity-30 dark:bg-blue-700 dark:bg-opacity-25">
                    Typescript
                  </div>
                  <div class="mr-4 mb-4 py-2 px-4 | text-sm | font-bold | whitespace-nowrap | rounded-full | border border-pink-500 dark:border-blue-300 | bg-pink-50 bg-opacity-30 dark:bg-blue-700 dark:bg-opacity-25">
                    Javascript
                  </div>
                  <div class="mr-4 mb-4 py-2 px-4 | text-sm | font-bold | whitespace-nowrap | rounded-full | border border-pink-500 dark:border-blue-300 | bg-pink-50 bg-opacity-30 dark:bg-blue-700 dark:bg-opacity-25">
                    jQuery
                  </div>
                  <div class="mr-4 mb-4 py-2 px-4 | text-sm | font-bold | whitespace-nowrap | rounded-full | border border-pink-500 dark:border-blue-300 | bg-pink-50 bg-opacity-30 dark:bg-blue-700 dark:bg-opacity-25">
                    i18n
                  </div>
                  <div class="mr-4 mb-4 py-2 px-4 | text-sm | font-bold | whitespace-nowrap | rounded-full | border border-pink-500 dark:border-blue-300 | bg-pink-50 bg-opacity-30 dark:bg-blue-700 dark:bg-opacity-25">
                    Java Portlets
                  </div>
                  <div class="mr-4 mb-4 py-2 px-4 | text-sm | font-bold | whitespace-nowrap | rounded-full | border border-pink-500 dark:border-blue-300 | bg-pink-50 bg-opacity-30 dark:bg-blue-700 dark:bg-opacity-25">
                    LESS
                  </div>
                </div>
              </div>
              <!---->
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
  <section class="section-container contact-me">
    <div class="max-w-screen-lg | flex justify-center | my-0 mx-auto">
      <div class="grid grid-cols-1 md:grid-cols-2 gap-6">
        <div class="flex flex-col justify-center | mb-12 md:mb-0">
          <h3 class="mb-4 | section-title contact-me__title">
            Just say hi
          </h3>
          <p class="mb-14 mr-0 md:mr-8 md:mb-20 | font-base">
            I'm always open to discuss about projects and new challenges, do not hesitate to contact me and let's talk.
          </p>
          <div class="flex flex-col items-start"><span class="mb-2 | text-xs">
            Email me at:
          </span> <a href="mailto:hi@christian-bravo.dev" class="font-bold | font-base">
              hi@christian-bravo.dev
            </a></div>
        </div>
        <section class="flex justify-center items-center">
          <div class="grid gap-6 grid-cols-4 md:grid-cols-2 lg:grid-cols-4 page-social-icons"><a target="_blank" href="https://codepen.io/christian-bravo" class="w-8 h-8 md:w-12 md:h-12 | p-1 | bg-transparent dark:text-gray-50 dark-hover:bg-blue-500 dark-hover:border-blue-500 text-gray-900 hover:bg-pink-500 hover:text-gray-50 hover:border-pink-500 rounded-full social-icon"><svg></svg></a><a target="_blank" href="https://github.com/christian-bravo7" class="w-8 h-8 md:w-12 md:h-12 | p-1 | bg-transparent dark:text-gray-50 dark-hover:bg-blue-500 dark-hover:border-blue-500 text-gray-900 hover:bg-pink-500 hover:text-gray-50 hover:border-pink-500 rounded-full social-icon"><svg></svg></a><a target="_blank" href="https://www.instagram.com/christian.bravo_/" class="w-8 h-8 md:w-12 md:h-12 | p-1 | bg-transparent dark:text-gray-50 dark-hover:bg-blue-500 dark-hover:border-blue-500 text-gray-900 hover:bg-pink-500 hover:text-gray-50 hover:border-pink-500 rounded-full social-icon"><svg></svg></a><a target="_blank" href="https://www.linkedin.com/in/christian151-bravo/" class="w-8 h-8 md:w-12 md:h-12 | p-1 | bg-transparent dark:text-gray-50 dark-hover:bg-blue-500 dark-hover:border-blue-500 text-gray-900 hover:bg-pink-500 hover:text-gray-50 hover:border-pink-500 rounded-full social-icon"><svg></svg></a></div>
        </section>
      </div>
    </div>
  </section>
</section>
`;
